tmt.spec

```diff
@@ -64,6 +64,7 @@ Obsoletes:      tmt-provision-virtual < %{version}-%{release}
 Obsoletes:      tmt-testcloud < 0.17
 Requires:       tmt == %{version}-%{release}
 Requires:       python3-testcloud >= 0.9.10
+Requires:       libvirt-daemon
 Requires:       libvirt-daemon-config-network
```

Comment:
Isn't it redundant to have the two libvirt packages here, since they are required by python3-testcloud?

Reply:
Yeah, just a workaround. Since `fedora-39` the `libvirt-daemon-config-network` does not require the daemon. I've checked this with @frantisekz today and he confirmed he's going to add this to `python3-testcloud` requires.

Reply:
Moved in 1462b842.